easybuild/easyconfigs/a/AFNI/AFNI-21.11.07-foss-2020a-R-3.6.3-Python-3.8.2.eb

```diff
@@ -0,0 +1,71 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Erica Bianco <erica.bianco@hpcnow.com>, Danilo Gonzalez, Maicon Faria
+# License::     GPL-v3.0
+# #
+easyblock = 'ConfigureMake'
+
+name = 'AFNI'
+version = '21.11.07'
+
+homepage = 'http://afni.nimh.nih.gov/'
+description = """AFNI is a set of C programs for processing, analyzing, and displaying functional MRI (FMRI) data -
+ a technique for mapping human brain activity."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'openmp': True, 'pic': True}
+
+versionsuffix = '-R-%(rver)s-Python-%(pyver)s'
+
+source_urls = ['https://afni.nimh.nih.gov/pub/dist/tgz/']
+sources = [{
+    'download_filename': '%(namelower)s_src.tgz',
+    'filename': SOURCELOWER_TGZ
+}]
+patches = ['%(name)s-%(version)s-Makefile.patch']
+checksums = [
+    '955e703d751c0179be2a3d88f46ee03b0e5c6619fbeea1617ec7afd2e8a31e16',  # afni-21.11.07.tgz
```

Comment:
@kErica could you check the checksum please?
On several systems, I see `c436920f5a1c5fc019f463773d0ce65c7763b3a686a28ff73319a2dabfc17adf`

Reply:
I checked it. It seems the developper changes the source file with the name I used regularly - no versioning of the sourcefile.
I change the source url and version to the one they have on their github project - more reproducible